Deal with cc -version reporting "2.7.2.1" or any other junk following the
third number.

Inspired by a larger patch from <jhs@freebsd.org> and apparently
Andreas Klemm <andreas@klemm.gtn.com>

--- configure.dist	Mon Oct 14 10:17:10 1996
+++ configure	Mon Oct 14 10:18:20 1996
@@ -446,7 +446,7 @@
 {
     app=$1; shift
     eval `$app -v 2>&1 | \
-	sed -n -e '/version/s/.* \([0-9]*\)\.\([0-9]*\).\([0-9]*\)/GCCdist=\1;GCCmajor=\2;GCCminor=\3/p'`
+	sed -n -e '/version/s/.* \([0-9]*\)\.\([0-9]*\).\([0-9]*\).*/GCCdist=\1;GCCmajor=\2;GCCminor=\3/p'`
     GCCversion="${GCCdist}.${GCCmajor}.${GCCminor}"; export GCCversion
     required="$1$2$3"
     actual="${GCCdist}${GCCmajor}${GCCminor}"
